### PR TITLE
add link status check for markdown files

### PR DESCRIPTION
**Pull request by Shreyas-21**

### CODE REVIEW

File: tool/olinkcheck/bin/utils.ml
Comment (cuihtlauac) [inline tool/olinkcheck/bin/utils.ml:1]: Remove this file

File: tool/olinkcheck/bin/main.ml
Comment (cuihtlauac) [inline tool/olinkcheck/bin/main.ml:1]: Remove this file

File: ocamlorg.opam.template [hunk 1]
```diff
@@ -1,4 +1,5 @@
 pin-depends: [
   ["tailwindcss.dev" "git+https://github.com/tmattio/opam-tailwindcss#7ef5bebbf48958370051b2368a4dd57b69627cf6"]
   ["hilite.dev" "git+https://github.com/tmattio/hilite#019d8a94d1981af19ca8d72e3d0bfde9dae6b284"]
+  ["olinkcheck.~dev" "git+https://github.com/tarides/olinkcheck#main"]
```
Comment (cuihtlauac) [inline ocamlorg.opam.template]:
```suggestion
  ["olinkcheck.~dev" "git+https://github.com/tarides/olinkcheck"]
```

For the time being, you don't need to specify the branch

File: ocamlorg.opam [hunk 2]
```diff
@@ -80,4 +81,5 @@ dev-repo: "git+https://github.com/ocaml/ocaml.org.git"
 pin-depends: [
   ["tailwindcss.dev" "git+https://github.com/tmattio/opam-tailwindcss#7ef5bebbf48958370051b2368a4dd57b69627cf6"]
   ["hilite.dev" "git+https://github.com/tmattio/hilite#019d8a94d1981af19ca8d72e3d0bfde9dae6b284"]
+  ["olinkcheck.~dev" "git+https://github.com/tarides/olinkcheck#main"]
```
Comment (cuihtlauac) [inline ocamlorg.opam]:
```suggestion
  ["olinkcheck.~dev" "git+https://github.com/tarides/olinkcheck"]
```

File: Makefile [hunk 3]
```diff
@@ -69,3 +69,7 @@ scrape: ## Generate the po files
 .PHONY: docker
 docker: ## Generate docker container
 	docker build -f Dockerfile . -t ocamlorg:latest
+
+.PHONY: linkcheck
+linkcheck:
+	opam exec -- dune exec --root . tool/olinkcheck/bin/main.exe md data
```
Comment (cuihtlauac) [inline Makefile]:
```suggestion
	opam exec -- dune exec --root . olinkcheck md data
```

**Review by cuihtlauac — DISMISSED** [review]

**Review by sabine — APPROVED** [review]

**Pull request merged**